static/app/gettingStartedDocs/apple/apple-ios.tsx

```diff
@@ -133,138 +243,241 @@ const getVerifyMetricsSnippetObjC = () => `
+              },
+            ],
+          },
+        ],
+  configure: params =>
+    isAutoInstall(params)
+      ? [
+          {
+            type: StepType.CONFIGURE,
+            description: t(
+              'The Sentry wizard will automatically patch your application:'
+            ),
+            configurations: [
+              {
+                description: (
+                  <List symbol="bullet">
+                    <ListItem>
+                      {t('Install the Sentry SDK via Swift Package Manager or Cocoapods')}
+                    </ListItem>
+                    <ListItem>
+                      {tct(
+                        'Update your [appDelegate: AppDelegate] or SwiftUI App Initializer with the default Sentry configuration and an example error',
+                        {
+                          appDelegate: <code />,
+                        }
+                      )}
+                    </ListItem>
+                    <ListItem>
+                      {tct(
+                        'Add a new [phase: Upload Debug Symbols] phase to your [xcodebuild: xcodebuild] build script',
+                        {
+                          phase: <code />,
+                          xcodebuild: <code />,
+                        }
+                      )}
+                    </ListItem>
+                    <ListItem>
+                      {tct(
+                        'Create [sentryclirc: .sentryclirc] with an auth token to upload debug symbols (this file is automatically added to [gitignore: .gitignore])',
+                        {
+                          sentryclirc: <code />,
+                          gitignore: <code />,
+                        }
+                      )}
+                    </ListItem>
+                    <ListItem>
+                      {t(
+                        "When you're using Fastlane, it will add a Sentry lane for uploading debug symbols"
+                      )}
+                    </ListItem>
+                  </List>
+                ),
+                additionalInfo: tct(
+                  'Alternatively, you can also [manualSetupLink:set up the SDK manually].',
+                  {
+                    manualSetupLink: (
+                      <ExternalLink href="https://docs.sentry.io/platforms/apple/guides/ios/manual-setup/" />
+                    ),
+                    stepsBelow: <strong />,
+                  }
+                ),
+              },
+            ],
+          },
+        ]
+      : [
+          {
+            type: StepType.CONFIGURE,
+            description: (
+              <p>
                 {tct(
-                  'Add a new [phase: Upload Debug Symbols] phase to your [xcodebuild: xcodebuild] build script',
+                  'Make sure you initialize the SDK as soon as possible in your application lifecycle e.g. in your AppDelegate [appDelegate: application:didFinishLaunchingWithOptions] method:',
```

Comment:
```suggestion
                  'Make sure you initialize the SDK as soon as possible in your application lifecycle e.g. in your UIAppDelegate: [appDelegate: - [UIAppDelegate application:didFinishLaunchingWithOptions:]] method:',
```
`- [UIAppDelegate application:didFinishLaunchingWithOptions:]` is the canonical way to refer to an ObjC instance method.

Reply:
Does this apply to the macOS getting started as well since I just copied pasted this from there.